packages/lesswrong/lib/index.ts

```diff
@@ -4,9 +4,14 @@ import './vulcan-users/index';
 
 // schema utils
 import './utils/schemaUtils'
+
+// collection utils
+import './collectionUtils'
+
 // Permissions
 import './permissions';
 
+
```

Comment:
Undo the changes to this file now that we've moved the SimpleSchema.extendOptions elsewhere